internal/store/persistentvolume.go

```diff
@@ -349,6 +352,45 @@ func persistentVolumeMetricFamilies(allowAnnotationsList, allowLabelsList []stri
 				}
 			}),
 		),
+		*generator.NewOptInFamilyGenerator(
+			descPersistentVolumeCSIAttributesName,
+			descPersistentVolumeCSIAttributesHelp,
+			metric.Gauge,
+			basemetrics.ALPHA,
+			"",
+			wrapPersistentVolumeFunc(func(p *v1.PersistentVolume) *metric.Family {
+				if p.Spec.CSI == nil {
+					return &metric.Family{
+						Metrics: []*metric.Metric{},
+					}
+				}
+
+				var csiMounter, csiMapOptions string
+				for k, v := range p.Spec.PersistentVolumeSource.CSI.VolumeAttributes {
+					if k == "mapOptions" {
+						csiMapOptions = v
+					} else if k == "mounter" {
+						csiMounter = v
+					}
```

Comment:
could you document where these attributes are coming from for future reference?

Reply:
added: "// storage attributes handled by external CSI drivers"